src/sxastarter/src/assets/sass/components/layout-container/_media.scss

```diff
@@ -0,0 +1,42 @@
+// Breakpoints variables
+$breakpoints-sm: 370px;
+$breakpoints-md: 768px;
+$breakpoints-lg: 1024px;
+$breakpoints-xl: 1280px;
+$breakpoints-xxl: 1536px;
+
+// Media queries
+@media only screen and (max-width: $breakpoints-sm) {
```

Comment:
currently you hardcoded @media queries for each breakpoint, making it impossible to introduce new ones by customer. You should iterate over a breakpoints map

Reply:
Removed!